pkg/apiserver/apiserver.go

```diff
@@ -236,12 +237,12 @@ func runE(flags *flags, log logr.Logger) error {
 
 	accountServer := v1.NewAccountServiceServer(accountClient)
 	apiserverv1.RegisterAccountServiceServer(grpcServer, accountServer)
-	if err := apiserverv1.RegisterAccountServiceHandlerServer(ctx, grpcGatewayMux, accountServer); err != nil {
+	if err := apiserverv1.RegisterAccountServiceHandler(ctx, grpcGatewayMux, grpcClient); err != nil {
```

Comment:
when using the JSON REST API calls it skips the gRPC server middlewares handling auth & other goodies.